### PR TITLE
Fixed grouping of EB boxes

### PR DESCRIPTION
**Pull request by IbrahimDev00**
**Changes made:** 
- Removed animation from the boxes for better UX
- Added proper margins and padding for each box
- Added a margin for the  "Executive body" header tag and moved it below the tabs section for better visibility.

**Additional fixes:**
- The box sizing purely depends on the images contained within, if the image aspect ratios are set right the box sizing will be taken care of.

### CODE REVIEW

File: public/Sampath.jpgZone.Identifier
Comment (vvs-basanth) [inline public/Sampath.jpgZone.Identifier:1]: Remove zone.identifier files

File: src/components/eb.tsx
Comment (vvs-basanth) [inline src/components/eb.tsx:1]: The images are not visible

**Review by vvs-basanth — CHANGES_REQUESTED** [review]
1) Remove all the Zone.identifiers files in the public folder
2) Images are not loading in the local host

Please try to create a branch and then make the changes or atleast make sure your remote repo is up to date with the upstream master branch.

File: src/components/eb.tsx
Comment (IbrahimDev00) [reply src/components/eb.tsx:1]: I reduced the Image sizes till we get new ones. For now the focus was on box alignments/positionining

Comment (vvs-basanth) [reply src/components/eb.tsx:1]: > I reduced the Image sizes

okay then!

**Comment by IbrahimDev00** [comment]
All the changes have been made, and all the latest commits have been added to this PR. 
Please od review it @vvs-basanth

**Comment by vvs-basanth** [comment]
Text content does not match server-rendered HTML
Look into this issue while hosting it locally
[https://react.dev/reference/react-dom/client/hydrateRoot](https://react.dev/reference/react-dom/client/hydrateRoot)

- Check with all the nesting elements once again
- Interactive elements cannot be nested atleast accroding to [https://html.spec.whatwg.org/#interactive-content-2](https://html.spec.whatwg.org/#interactive-content-2)

Possible way (atleat what I can think off)
1) Use `useEffect`
2) Disable pre-rendering
3) If that doesnt solve then supress the hydration warning and we can go ahead with this.

**Comment by vvs-basanth** [comment]
Well @IbrahimDev00 
I have gone through this and I found out that the issue I am facing is over the Firefox Browser and not over Chrome
Everything was fine with Chrome.

image

Well this must be related to SSR which cannot work if there is improper nesting elements.
But Chrome doesnt care about that
The components are getting rendered but does not match the client side HTML script.

Take your time on this
I am assigning it to P3 (Moderate)

**Comment by IbrahimDev00** [comment]
> Well @IbrahimDev00 I have gone through this and I found out that the issue I am facing is over the Firefox Browser and not over Chrome Everything was fine with Chrome.
> 
> image
> 
> Well this must be related to SSR which cannot work if there is improper nesting elements. But Chrome doesnt care about that The components are getting rendered but does not match the client side HTML script.
> 
> Take your time on this I am assigning it to P3 (Moderate)

Please check the latest commit. It should be fixed.

**Pull request merged**